build.ps1

```diff
@@ -57,6 +58,7 @@ $ArchX86 = @{
   PlatformInstallRoot = "$BinaryCache\x86\Windows.platform\";
   SDKInstallRoot = "$BinaryCache\x86\Windows.platform\Developer\SDKs\Windows.sdk";
   XCTestInstallRoot = "$BinaryCache\x86\Windows.platform\Developer\Library\XCTest-development";
+  ToolchainInstallRoot = "$BinaryCache\x86\unknown-Asserts-development.xctoolchain";
```

Comment:
This should be removed.  I do not expect to ever support a 32-bit toolchain, that will always require cross-compilation from a 64-bit toolchain.  The 32-bit targets are where you can run code only.

Reply:
Makes sense! I removed it in latest.